### PR TITLE
docs: Feedback from #15613. NFC

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: site/source/docs/compiling/Building-Projects.rst [hunk 1]
```diff
@@ -261,19 +261,19 @@ In some cases it makes sense to modify the build scripts so that they build the
...
-library, Emscripten will produce an ``.so`` library that is actually just a
-regular ``.o`` object file (Under the hood it uses `ld -r` to combine objects
-into a single larger object).  When these faux "shared libraries" are linked
-into your application they are effectively linked as static libraries.  When
-building these shared libraries *Emcc* will ignore other shared libraries on the
-command line.  This is to ensure that the same dynamic library is not linked
-multiple times in intermediate build stages, which would result in duplicate
-symbol errors.
+Emscripten's goal is to generate the fastest and smallest possible code. For
+that reason it focuses on compiling an entire project into the single file,
```
Comment (dschuff) [inline site/source/docs/compiling/Building-Projects.rst:265]:
```suggestion
that reason it focuses on compiling an entire project into a single wasm file,
```

**Review by dschuff — APPROVED** [review]

File: site/source/docs/compiling/Building-Projects.rst (hunk 1 above)
Comment (sbc100) [reply site/source/docs/compiling/Building-Projects.rst:265]: Its really a JS file + a wasm file..  I was trying to find a way to keep the meaning without having to go into those details.

Comment (dschuff) [reply site/source/docs/compiling/Building-Projects.rst:265]: hm, true. maybe "single module" is more correct.

Comment (sbc100) [reply site/source/docs/compiling/Building-Projects.rst:265]: Yup that works.

**Pull request merged**